Automattic-Tracks-iOSTests/Test Helpers/Mocks.swift

```diff
@@ -11,21 +11,24 @@ enum MockError: Error {
 
 class MockEventLoggingDataSource: EventLoggingDataSource {
     private(set) var loggingEncryptionKey: String
-    private(set) var previousSessionLogPath: URL?
-    private(set) var currentSessionLogPath: URL?
     private(set) var logUploadURL: URL
     private(set) var logUploadQueueStorageURL: URL
     private(set) var loggingAuthenticationToken: String = ""
+    private let sessionLogPath: URL?
 
     init(
         encryptionKey: String = "foo",
         sessionLogPath: URL? = nil,
         logUploadUrl: URL = URL(string: "example.com")!,
         queueUrl: URL = FileManager.default.documentsDirectory.appendingPathComponent(UUID().uuidString)) {
-        loggingEncryptionKey = encryptionKey
-        previousSessionLogPath = sessionLogPath
-        logUploadURL = logUploadUrl
-        logUploadQueueStorageURL = queueUrl
+        self.loggingEncryptionKey = encryptionKey
```

Comment:
This are all just changed to use `self` because I had to for sessionLogPath to disambiguate and I wanted it to be consistent.